test_cases/README.md

```diff
@@ -0,0 +1,24 @@
+## Regression tests for typeshed
+
+This directory contains regression tests for the stubs found elsewhere in the
+typeshed repo. Each file contains a number of test cases, all of which should
+pass a type checker without error.
```

Comment:
In the long term we should probably also allow test that fail a type checker to be able to test the negative case.

Reply:
Yup. Once we're running mypy on this we can do that by `# type: ignor`ing (with specific error codes) lines that we want to fail and running mypy with `--warn-unused-ignores`.